Exiled.Example/Example.cs

```diff
@@ -92,6 +92,7 @@ private void RegisterEvents()
             Exiled.Events.Handlers.Player.Shooting += playerHandler.OnShooting;
             Exiled.Events.Handlers.Player.ReloadingWeapon += playerHandler.OnReloading;
             Exiled.Events.Handlers.Player.ReceivingEffect += playerHandler.OnReceivingEffect;
+            Exiled.Events.Handlers.Player.TriggeringTesla += playerHandler.OnTriggeringTesla;
```

Comment:
You never unregister this event.

Also I don't see the point of any of the changes in Exiled.Example, I feel there's already more than enough code examples for people to get an idea of what to do.

Reply:
ups sorry, never mean for this to got to the pr, just the correction of the event, experimenting with pr rn

Reply:
I don't actually know how to do PR efficienly, If I need to delete the fork and make another one with just the UsedItem correction I can do it.